app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -48,6 +48,9 @@ export default function () {
         this.route("colors", function () {
           this.route("show", { path: "/:scheme_id" });
         });
+        this.route("font", function () {
```

Comment:
```suggestion
        this.route("fonts", function () {
```

Would prefer the plural form pls

Reply:
Also this should be in the /admin/config/fonts route namespace, the `customize` one is old and no longer supported